### DISCUSSION TITLE
How to set striped row off?

### DISCUSSION THREAD

**Discussion by andresvennera** [Power-Components/livewire-powergrid#631] [Q&A]
I am using Powergrid with Bootstrap. By default the table type is striped (grey and white), where can I remove that setting for a given powergrid? Thank you very much for your help!

**Comment by abhaypithadiya** [comment]
Hey! You can go into \vendor\power-components\livewire-powergrid\src\Themes\Bootstrap5.php and remove the 'table-striped' from the table function. Hope this helps!

**Reply by andresvennera** [reply]
Thank you very much! It was exactly what I was looking for. Regards!

**Reply by akulmehta** [reply]
@abhaypithadiya  and @andresvennera this is not the ideal way of removing the table-striped. The reason being, if you run composer update and there is any new version of PowerGrid, your changes will be lost. Changing anything in the vendor folder is not good practice. 

Alternatively you can put your own styling by publishing views.

To do this you first publish your views using

```
php artisan vendor:publish --tag=livewire-powergrid-views --force && php artisan view:clear
```

This will publish the vendor views to your own laravel `resources/views` folder, where you can customize it. 

Then you can override the `resources/views/vendor/livewire-powergrid/components/frameworks/bootstrap5/table-base.blade.php` file to something like this

```php
<div>
    <div class="col-md-12">
        @include($theme->layout->header, [
                'enabledFilters' => $enabledFilters
        ])
    </div>
    <div class="table table-bordered table-hover table-checkable table-highlight-head mb-2" style="{{ $theme->table->divStyle }}">
        @include($table)
    </div>
    <div class="row">
        <div class="col-12 overflow-auto">
            @include($theme->footer->view, ['theme' => $theme])
        </div>
    </div>
</div>

```

**Reply by abhaypithadiya** [reply]
Hey! Thankyou so much for this information. Will use this now onwards!

**Reply by andresvennera** [reply]
Thanks Akul Mehta for the explanation!
